docs/quickstarts/outlook-quickstart-json-manifest.md

```diff
@@ -0,0 +1,271 @@
+    ```
+
+### Explore the project
+
+The add-in project that you've created with the Yeoman generator contains sample code for a very basic task pane add-in.
+
+- The **./manifest/manifest.json** file in the root directory of the project defines the settings and capabilities of the add-in.
+- The **./src/taskpane/taskpane.html** file contains the HTML markup for the task pane.
+- The **./src/taskpane/taskpane.css** file contains the CSS that's applied to content in the task pane.
+- The **./src/taskpane/taskpane.ts** file contains code that calls the Office JavaScript library to facilitate interaction between the task pane and Outlook.
+- The **./src/command/command.html** file will be edited by WebPack at build time to insert an HTML `<script>` tag that loads the JavaScript file that is transpiled from the command.ts file.
+- The **./src/command/command.ts** file has little code in it at first. Later in this article, you'll add code to it that calls the Office JavaScript library and that executes when a custom ribbon button is selected.
+
+### Update the code
+
+1. Open your project in VS Code or your preferred code editor.
+
+    > [!TIP]
+    > On Windows, you can navigate to the root directory of the project via the command line and then enter `code .` to open that folder in VS Code. 
+
+1. Open the file **./src/taskpane/taskpane.html** and replace the entire **\<main\>** element (within the **\<body\>** element) with the following markup. This new markup adds a label where the script in **./src/taskpane/taskpane.ts** will write data.
+
+    ```html
+    <main id="app-body" class="ms-welcome__main" style="display: none;">
+        <h2 class="ms-font-xl"> Discover what Office Add-ins can do for you today! </h2>
+        <p><label id="item-subject"></label></p>
+        <div role="button" id="run" class="ms-welcome__action ms-Button ms-Button--hero ms-font-xl">
+            <span class="ms-Button-label">Run</span>
+        </div>
+    </main>
+    ```
+
+1. In your code editor, open the file **./src/taskpane/taskpane.ts** and add the following code within the **run** function. This code uses the Office JavaScript API to get a reference to the current message and write its **subject** property value to the task pane.
+
+    ```typescript
+    // Get a reference to the current message.
+    let item = Office.context.mailbox.item;
+
+    // Write a message property value to the task pane.
+    document.getElementById("item-subject").innerHTML = "<b>Subject:</b> <br/>" + item.subject;
+    ```
+
+### Try it out
+
+[!INCLUDE [alert use https](../includes/alert-use-https.md)]
+
+1. Run the following command in the root directory of your project. When you run this command, the local web server starts and your add-in will be [sideloaded](../outlook/sideload-outlook-add-ins-for-testing.md). 
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+1. Use the Classic Ribbon in Outlook. The remainder of these instructions assume this.  
```

Comment:
```suggestion
1. Use the classic ribbon in Outlook. The remainder of these instructions assume this.  
```
"classic ribbon" isn't capitalized. If you want to emphasize it, add bold format.